static/openapi/prerendering-spec.json

```diff
@@ -1989,11 +1987,195 @@
         }
       }
     },
-    "/api/prerender/v1/origins/{_key}": {
+    "/api/prerender/v1/origins/{origin}": {
+      "get": {
+        "summary": "Get origin",
+        "tags": ["Origins"],
+        "description": "This endpoint returns an origin from the origin-related GDN fabric.",
+        "parameters": [
+          {
+            "schema": { "type": "string", "minLength": 1 },
+            "in": "path",
+            "name": "origin",
+            "required": true,
+            "description": "Origin hostname"
+          },
+          {
+            "schema": { "type": "string" },
+            "in": "header",
+            "name": "x-photoniq-customerid",
+            "required": true,
+            "description": "Customer unique identifier"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful response",
```

Comment:
@joubert-macrometa , should this be "Success response"?

Reply:
@sandro-macrometa it looks like to me all the 200 responses still say "Succesful response."
Could you fix these to tell what was successful?  I have a suggestion above how you can fix this.

Reply:
@sandro-macrometa disreguard.  I just saw the message below.  All good.

Reply:
@joubert-macrometa, I know how to fix this, but this would require a refactor because we are reusing some common schema objects to avoid repetition. The `Successful response` schema is one of them.